app/src/main/java/com/google/droidjump/models/Bat.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump.models;
+
+import android.content.res.Resources;
+import android.graphics.BitmapFactory;
+import com.google.droidjump.R;
+
+public class Bat extends Obstacle {
+
+    public Bat(int x, int y, Resources resources) {
+        super(x, y, R.mipmap.bat, resources);
+        this.y = (y - picture.getHeight());
```

Comment:
nit: this computation seems to be repeated in 2 place. Maybe extract in a common method?

Reply:
Done

Reply:
Done